scripts/reinitialize-aurora.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+# This shell script reinitializes the aurora test database.
+# The environment variable AURORA_CONNECTION_STRING (typically set in
+# .envrc.local) is assumed to hold the authenticated connection string to the
+# writable aurora instance to act on.
+
+# In order to avoid accidental loss of data the database is assumed to already
+# be suitably empty.
```

Comment:
Perhaps we can check for this and bail out early with an error if it's false.

Reply:
Good idea. Shoudln't be too hard.

Reply:
Done~